solvebio/cli/data.py

```diff
@@ -460,6 +460,41 @@ def import_file(args):
     return imports, dataset
 
 
+def download(args):
+    """
+    Given a folder or file, download all the files contained within it (not recursive).
+    """
+    local_folder_path = args.local_path
+    sb_path = args.full_path
+    query = args.query
+
+    base_remote_path, path_dict = Object.validate_full_path(sb_path)
+
+    sb_object = Object.get_by_full_path(base_remote_path)
+
+    # If user sets the home directory, expand the path
+    local_folder_path = os.path.expanduser(local_folder_path) if local_folder_path.startswith(
+        '~') else local_folder_path
+
+    if sb_object.is_file:
+        sb_object.download(local_folder_path)
+        print('Downloaded: {}'.format(sb_object.full_path))
+
+    elif sb_object.is_folder:
+        files = sb_object.files(query=query) if query else sb_object.files()
```

Comment:
to be consistent with the other commands, maybe we should change this to use the same `glob=` syntax that the `upload` command and `tag` command full_path use?

thoughts @davecap ?

Reply:
Yes I think we should use `glob` if possible.